message_definitions/v1.0/common.xml

```diff
@@ -3465,7 +3474,7 @@
       <field type="uint16_t" name="cog" units="cdeg">Course over ground (NOT heading, but direction of movement) in degrees * 100, 0.0..359.99 degrees. If unknown, set to: UINT16_MAX</field>
       <field type="uint8_t" name="satellites_visible">Number of satellites visible. If unknown, set to 255</field>
       <field type="uint8_t" name="dgps_numch">Number of DGPS satellites</field>
-      <field type="uint32_t" name="dgps_age">Age of DGPS info</field>
+      <field type="uint32_t" name="dgps_age" units="cs">Age of DGPS info</field>
```

Comment:
This is a weird unit for this, (also to late to change it now, but honestly uint32_t is almost a waste of payload space, RTK corrections are going to be worthless after 497 hours :) Could we treat it as a ms field at least just given that its a much more common unit and we have more then enough space for it.